test/v1/service.go

```diff
@@ -201,9 +202,16 @@ func WaitForServiceLatestRevision(clients *test.Clients, names test.ResourceName
 // Service returns a Service object in namespace with the name names.Service
 // that uses the image specified by names.Image.
 func Service(names test.ResourceNames, fopt ...rtesting.ServiceOption) *v1.Service {
-	a := append([]rtesting.ServiceOption{
-		rtesting.WithConfigSpec(*ConfigurationSpec(pkgTest.ImagePath(names.Image))),
-	}, fopt...)
+	var a []rtesting.ServiceOption
+	if len(names.Containers) != 0 {
```

Comment:
Should we augment multiple containers via the ServiceOptions rather than doing this? Containers aren't really names I think.